canonical-web-and-design/jaas-dashboard
# Render JAAS instead of the controller UUID

## Done
For the JAAS controller, render "JAAS" instead of its UUID.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View your models on all three groupings. The controller column should say "JAAS"

## Details

Fixes #313
